compiler/src/main/scala/org/scalajs/core/compiler/JSDefinitions.scala

```diff
@@ -127,6 +128,10 @@ trait JSDefinitions { self: JSGlobalAddons =>
     lazy val BoxesRunTime_boxToCharacter = getMemberMethod(BoxesRunTimeModule, newTermName("boxToCharacter"))
     lazy val BoxesRunTime_unboxToChar    = getMemberMethod(BoxesRunTimeModule, newTermName("unboxToChar"))
 
+    lazy val compilerInternalAnnotations = List(
+      RawJSTypeAnnot, JSFullNameAnnotation, SJSDefinedAnonymousClassAnnotation,
+      WasPublicBeforeTyperAnnotation, ExposedJSMemberAnnot
+    )
```

Comment:
This should be defined in `PrepJSInterop`. (also 4 spaces, and `)` at the end of the previous line)